code/src/components/NotFound.js

```diff
@@ -0,0 +1,12 @@
+import  React from 'react'
+
+
+
+
+
+export const NotFound =()=>{
```

Comment:
Really inspiring to see that you have reached this step in red level! Something for us to aim for next time :)